action/receipt.go

```diff
@@ -30,6 +30,7 @@ type (
 		BlobGasPrice       *big.Int
 		ContractAddress    string
 		TxIndex            uint32
+		EffectiveGasPrice  *big.Int
```

Comment:
why we need to add this into receipt? like caller needs to use it do some calculation?